tests/utils.py

```diff
@@ -803,6 +803,54 @@ def datetime_primitives1(cls, session: "Session") -> DataFrame:
         )
         return session.create_dataframe(data, schema)
 
+    @classmethod
+    def time_primitives1(cls, session: "Session") -> DataFrame:
```

Comment:
Where are these being used?

Reply:
it's now used in the first case of the test_to_time